src/Home.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  ScrollView,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  Platform,
+  SafeAreaView,
+} from 'react-native';
+import styles from './styles';
+
+import DaySelector from './components/DaySelector';
+import Meals from './components/Meals';
+import { MealCategories, MealText, Category, AppButton, Day } from './models/meals.model';
+
+const Home: React.FC = () => {
+  const [selectedDay, setSelectedDay] = useState<Day | null>(null);
+
+  const homeButtons: AppButton[] = [
+    { id: 'recipes', label: 'Recipes', onButtonPress: () => console.log('recipes') },
+    { id: 'chatGPT', label: 'Consult with chatGPT', onButtonPress: () => console.log('chatGPT') },
+  ];
+
+  const handleBreakfastText = (text: MealText, category: Category) => {
+    console.log('text, category', text, category);
+  };
+
+  const handleDayPress = (day: Day): void => {
+    setSelectedDay(day);
+  };
+
+  const getDay = (day: Day): string => {
+    return day;
+  };
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView>
+        <KeyboardAvoidingView
+          behavior={Platform.OS === 'android' ? 'height' : 'padding'}
+          style={styles.container}
+        >
+          <View style={styles.form}>
+            <DaySelector
+              daysOfWeek={Object.values(Day)}
+              selectedDay={selectedDay}
+              onSelectDay={handleDayPress}
+            />
+            <Text style={styles.title}>
+              {selectedDay ? `${getDay(selectedDay)} menu` : 'Select a day'}
+            </Text>
+            {selectedDay ? (
+              <Meals
+                handleBreakfastText={handleBreakfastText}
+                mealCategories={Object.values(MealCategories)}
```

Comment:
why do we need this prop? are u planning on having different meal categories in some stage?

Reply:
meal categories=breakfast, lunch, dinner.
 i want to keep track of each one of them

Reply:
so why not use it in the component directly instead of receiving it as a `prop`?

Reply:
i fixed that in the new branch i'm working on